src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String greeting;
+    private int variable;
+    private int age;
+    private int size;
+
+    public static void main(String[] args) {
+        Dog dog = new Dog();
+        Cat cat = new Cat();
+    }
+
+    private String getFromOneToTen() {
+        String result = "";
+
+        for (int i = 0; i < 10; i++) {
+            if (i % 2 == 0) {
+                result = result + i + " ";
+            }
+        }
+        return result;
+    }
+
+    private void sayHello(String helloMe) {
+        String one = "Hello, Mate! I am happy to see you! This class looks weird! ";
+        one = one + "Your task is to fix the checkstyle in this file.";
+        String two = " Your goal is to run `mvn clean package` and receive `BUILD SUCCESS` message";
```

Comment:
You can use direct concatenation
```suggestion
        String youreAwesomeVariableName = "Hello, Mate! I am happy to see you! This class looks weird! "
        		+ "Your task is to fix the checkstyle in this file."
        		+ " Your goal is to run `mvn clean package` and receive `BUILD SUCCESS` message";
```